packages/react/src/components/Button/BaseButton.tsx

```diff
@@ -482,7 +482,7 @@ export class BaseButton extends React.Component<IBaseButtonProps, IBaseButtonSta
 
   private _onRenderDescription = (props: IButtonProps) => {
     // eslint-disable-next-line deprecation/deprecation
-    const { secondaryText = this.props.description } = props;
```

Comment:
this.props.description is of type IStyle. No idea why it was ever allowed as a default.

Reply:
https://github.com/microsoft/fluentui/pull/4843 <-- seems to be from back then in case you were curious.

Reply:
It seems like the more correct fix is to change the type of `description` to `string` in `Button.types.ts`. It seems like there was a mistake in typing it as `IStyle` since the PR @SahilTara points out based on the usage of the prop in `BaseButton`.